tools/travis-before-install.sh

```diff
@@ -33,5 +33,9 @@ python -V
 popd
 
 pip install --upgrade pip setuptools
+# pycparser doesn't work with optimized Python builds that
+# strip out docstrings, so preemptively install pycparser
+# with PYTHONOPTIMIZE unset.
+PYTHONOPTIMIZE= pip install -v pycparser
```

Comment:
```suggestion
PYTHONOPTIMIZE="" pip install -v pycparser
```
I am daft when it comes to bash. Can you ass the `set -e` on the top as well? If this is fixed in newer pycparser versions, maybe you can change comment to reflect that it will be OK to delete the line again?